sidebars.ts

```diff
@@ -569,20 +569,16 @@ const sidebars: SidebarsConfig = {
       id: 'build/examples/index',
       label: '📚 Examples',
     },
+    {
+      type: 'doc',
+      id: 'build/sdks/python/index',
+      label: '📚 Python',
```

Comment:
Can we update the label with Python icon? Not sure if that is possible @mason-at-pieces @choir27